### ISSUE TITLE
Accessibility of images

### ISSUE THREAD

**Issue by martinthomson** [c2pa-org/specifications#39]
https://c2pa.org/specifications/specifications/1.3/specs/C2PA_Specification.html#_hashing_a_bmff_formatted_asset contains some images, which would appear to include a bunch of really important procedural information.  The information in these images is not very accessible.  I am having a lot of trouble reading the text and I consider myself as having relatively good eyesight.

**Comment by lrosenthol** [comment]
@martinthomson I was pretty sure all the images had associated AltText - but will most definitely make sure!  Thanks for pointing this out.

**Comment by martinthomson** [comment]
Note that I am also referring to the accessibility of the images for a sighted reader.  Alt text is one thing, but the content of the images seems to be something that should be expanded more - likely in the form of text - in order to fully understand the procedures that are involved.

**Comment by lrosenthol** [comment]
@martinthomson I have made sure that every image and figure in the spec has useful AltText.  Also, all images/figures are now linked into the text itself to help provided context.

I have an action item to work on some of images to make sure that there is associated REAL text to help explain them.

**Comment by lrosenthol** [comment]
This issue has been migrated to the c2pa-org/specs-core repository. Please continue the discussion at https://github.com/c2pa-org/specs-core/issues/2178
